MonstieWash/Assets/Scripts/UI/ScoreUIManager.cs

```diff
@@ -0,0 +1,137 @@
+using JetBrains.Annotations;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
+
+public struct PHTask
+{
+    public string name;
+    public bool complete;
+    public int score;
+
+    public PHTask(string _name, bool _complete, int _score)
+    {  name = _name; complete = _complete; score = _score; }
+}
+
+public class ScoreUIManager : MonoBehaviour
+{
+    [SerializeField] private Image m_clipboard;
+    [SerializeField] private GameObject m_lineItem;
+    [SerializeField] private TextMeshProUGUI m_totalScore;
+    [SerializeField] private GameObject m_strikeOut;
+
+    //Probably temporary, may change to calc scale with number of tasks later.
+    [SerializeField][Range(0.0f, 5.0f)] private float m_spacing;
+
+    /// <summary>
+    /// Placeholder. Suggest Json to player prefs.
+    /// </summary>
+    [SerializeField] private List<PHTask> m_tasks;
+    [SerializeField] private List<GameObject> m_LIList;
+
+    [SerializeField][Range(0.0f, 2.0f)] private float m_secondsWait = 1.0f;
+    private bool m_skip = false;
+
+    // Start is called before the first frame update
+    private void OnEnable()
+    {
+        MenuInputManager.Inputs.OnAltSelect += Inputs_OnAlt_Select;
+
+        m_tasks = PlaceholderTaskFill();
+
+        StartCoroutine(LineItemSetActive());
+    }
+
+    private void OnDisable()
+    {
+        MenuInputManager.Inputs.OnAltSelect -= Inputs_OnAlt_Select;
+    }
+
+    /// <summary>
+    /// Fills the task list and returns it.
+    /// </summary>
+    /// <returns>List of tasks with score multiplier</returns>
+    private List<PHTask> PlaceholderTaskFill()
+    {
+        List<PHTask> ph_tasks = new List<PHTask>();
+
+        for (int i = 0; i < 8; i++)
+        {
+            ph_tasks.Add(new PHTask("Task Name Here", true, 10));
+        }
+
+        return ph_tasks;
+    }
+
+    private IEnumerator LineItemSetActive()
+    {
+        Transform clipTransform = m_clipboard.transform;
+        RectTransform LITransform;
+
+        TextMeshProUGUI name = null;
+        TextMeshProUGUI score = null;
+        GameObject strikeOut = null;
+
+        int totalScore = 0;
+
+        for (int i = 0; i < m_tasks.Count; i++)
+        {
+            m_LIList.Add(Instantiate(m_lineItem, clipTransform));
+            m_LIList[i].SetActive(false);
+            LITransform = m_LIList[i].GetComponent<RectTransform>();
+            LITransform.anchoredPosition = new Vector2(0, LITransform.rect.height / m_spacing * -i);
+
+            name = m_LIList[i].transform.Find("TaskName").GetComponent<TextMeshProUGUI>();
+            strikeOut = m_LIList[i].transform.Find("StrikeOut").gameObject;
+
+            name.text = m_tasks[i].name;
+
+            m_LIList[i].SetActive(true);
+            strikeOut.SetActive(false);
+        }
+
+        for (int j = 0; j < m_LIList.Count; j++)
```

Comment:
This is entirely up to you if you want to change this but just thought I'd mention that since these two for loops are not nested you don't need to use a different named iterator variable. You can feel free to make them both `i` if you want.
oh also just remember to use var for all these loops as per convention
```suggestion
        for (var j = 0; j < m_LIList.Count; j++)
```